accepted/0000-custom-dev-dependencies.md

```diff
@@ -0,0 +1,76 @@
+# Custom devDependencies
+
+## Summary
+
+I would like to define different sets of devDependencies, so I can install different sets of dependencies for different types of builds (unit testing / e2e / production). Aside from 
+dependencies and devDependencies you would be able to define a custom set of packages to install for development. 
+
+## Motivation
+
+I want to build two different test containers, one for Karma, one for Cypress. I am installing Cypress now when I install my devDependencies, even if 
+I want to do Karma tests. Cypress takes a long time to install, so it's really increasing my build times. Maybe I could create different versions of 
+my package.json and copy the correct one, but that seems prone to errors. 
+
+## Detailed Explanation
+
+package.json can have multiple dependencies:
+
+```
+{
+	"version": "0.0.0",
+	"license": "MIT",
+	"angular-cli": {},
+	"scripts": {
+		// scripts
+	},
+	"private": true,
+	"dependencies": {
```

Comment:
Don't you mean `devDependencies`?

Reply:
No, because dependencies -> default would be equivalent to dependencies, where dependencies -> dev would be equivalent to devDependencies.